docs/add-root-ca.rst

```diff
@@ -0,0 +1,45 @@
+Add a root CA to the web browser
+================================
+
+.. todo::
+
+  The following test has been realized on vagrant only - One should list
+  all requirements for reaching the UI using an operationnal cluster.
+  Web browsers version used for tests:
+  Googe Chrome: Version 75.0.3770.80 | Mozilla Firefox: Version 67.0
+
+Run the following command as ``root`` on the bootstrap node:
```

Comment:
Missing command? (there is nothing after your `:` but another sentence that ends with `:`)

Reply:
Correction done.